src/config/workload.rs

```diff
@@ -0,0 +1,412 @@
+use super::*;
```

Comment:
This file is a bit of a hodgepodge. I think workload is complex enough to warrant some structure both in config and in code organization.

Reply:
Agree. I'd love to discuss potential ways that we can address this. As we support additional types of services / protocols / ... the complexity here goes up pretty significantly.

Reply:
Per discussion, #12 opened to follow-up on this